mne/io/reference.py

```diff
@@ -281,7 +281,7 @@ def set_bipolar_reference(inst, anode, cathode, ch_name=None, ch_info=None,
 
     Parameters
     ----------
-    inst : instance of Raw | Epochs |Evoked
+    inst : instance of Raw | Epochs | Evoked
```

Comment:
I think there were some reasons why we did this. We have some separation between the final fiff class and the subclasses.
Second the good way to test this would be to check for isinstance(inst, _BaseRaw)